src/IonParserBinaryRaw.ts

```diff
@@ -208,74 +211,26 @@ export class ParserBinaryRaw {
         return ParserBinaryRaw._readVarSignedIntFrom(this._in);
     }
 
-    private readVarLongInt() : LongInt {
-        let bytes = [];
-        let byte = this._in.next();
-        let isNegative = (byte & 0x40) !== 0;
-        let stopBit = byte & 0x80;
-        bytes.push(byte & 0x3F);  // clears the sign/stop bit
-        while(!stopBit) {
-            byte = this._in.next();
-            stopBit = byte & 0x80;
-            byte &= 0x7F;
-            bytes.push(byte);
-        }
-        return LongInt.fromVarIntBytes(bytes, isNegative);
-    }
-
-    static _readSignedIntFrom(input: BinarySpan, numberOfBytes: number) : LongInt {
+    static _readSignedIntFrom(input: BinarySpan, numberOfBytes: number) : SignAndMagnitudeInt {
```

Comment:
`SignAndMagnitudeInt` is a simple class that pairs a `magnitude: JSBI` with an `isNegative: boolean` for the sake of returning a single value representing a signed int that might be `-0`. It is only used internally.